tests/sv/climate_HassClimateSetTemperature.yaml

```diff
@@ -1,6 +1,49 @@
 language: sv
 tests:
-- sentences: []
-  intent:
-    name: HassClimateSetTemperature
-    slots: {}
+  - sentences:
+      - "sätt temperaturen till 30°"
+      - "ändra temperaturen till 30°"
```

Comment:
Kan man ens säga ° ? ;)

Reply:
I'm not sure how that is suppose to be handled if it is just verification that it is ignored in the parsing of sentences. It is based on the English version of the same test.

Reply:
If it's the same in the english file, it's okay with me. And I just realised I wrote my comment in Swedish :)

Reply:
I think they are just ignored as % seems to be.